common/lib/capa/capa/tests/test_input_templates.py

```diff
@@ -473,6 +503,19 @@ def test_message(self):
         xpath = "//span[@class='message']"
         self.assert_has_text(xml, xpath, self.context['msg'])
 
+    def test_description(self):
```

Comment:
These tests are basically checking accessibility. Shouldn't they be marked as a11y tests?